plasmapy/classes/simulation.py

```diff
@@ -45,7 +46,7 @@ class MHDSimulation():
     def __init__(self, plasma):
         """
         """
-        self.dt = 0.01 * u.s
+        self.dt = 1e-4 * u.s
```

Comment:
Do we intend to allow setting this via init arguments later on? Seems to me it would be applicable to only a few selected regimes otherwise, right?